sql/Pg-database.sql

```diff
@@ -487,6 +499,7 @@ INSERT INTO country(short_name,name) VALUES ('VN','Viet Nam');
 INSERT INTO country(short_name,name) VALUES ('VU','Vanuatu');
 INSERT INTO country(short_name,name) VALUES ('WF','Wallis and Futuna Islands');
 INSERT INTO country(short_name,name) VALUES ('WS','Samoa');
+INSERT INTO country(short_name,name) VALUES ('XK','Kosovo');
```

Comment:
Pg-database isn't loaded anymore on existing installs. Do we want these in existing installs too? Then I think we need to add a schema-change for them. (Those *are* being run on existing schemas.)